static/app/views/projects/projectContext.tsx

```diff
@@ -39,6 +39,7 @@ type Props = {
   projectId: string;
   orgId: string;
   children: ((props: ChildFuncProps) => React.ReactNode) | React.ReactNode;
+  loadingProjects: boolean;
```

Comment:
Can we make this boolean more explanatory (`shouldLoadProjects`, `isLoadingProjects`, etc.)

Reply:
Nevermind turns out this is a built-in thing with the Projects HOC, my bad